crates/config/src/rule.rs

```diff
@@ -76,6 +79,32 @@ impl<L: Language> RuleConfig<L> {
       MetaVarMatchers::default()
     }
   }
+
+  pub fn matches_path(&self, path: &Path) -> bool {
+    let mut glob_set_builder: GlobSetBuilder = GlobSetBuilder::new();
+    let mut found_glob_conditional = false;
+    if let Some(files) = &self.files {
+      for path in files {
+        if let Ok(glob) = Glob::new(path.as_str()) {
+          found_glob_conditional = true;
+          glob_set_builder.add(glob);
+        }
+      }
+    }
+    if let Some(files) = &self.ignores {
+      for path in files {
+        if let Ok(glob) = Glob::new(format!("{}{}", "!", path).as_str()) {
+          found_glob_conditional = true;
+          glob_set_builder.add(glob);
+        }
+      }
+    }
+    if found_glob_conditional {
+      return glob_set_builder.build().unwrap().is_match(path);
```

Comment:
I'd like to move this call into an attribute so the Globset isn't built every time this is evaluated. Plus, there's an unwrap here where it should fail with a nicer error at parse time. Any thoughts?

Reply:
Let's keep it here for now. I can refactor it later